content/en/integrations/guide/pivotal-cloud-foundry-manual-setup.md

```diff
@@ -0,0 +1,502 @@
+---
+title: Pivotal Cloud Foundry Manual Setup Guide
+kind: guide
```

Comment:
If this is a guide, can you update the [Integrations Guides index page](https://docs.datadoghq.com/integrations/guide/)?